Kiosk/Bid Fulfillment/ManualCreditCardInputViewController.swift

```diff
@@ -36,34 +36,34 @@ class ManualCreditCardInputViewController: UIViewController, RegistrationSubCont
         billingZipWrapperView.isHidden = true
 
         // We show the enter credit card number, then the date switching the views around
-        viewModel
-            .cardFullDigits
+        cardNumberTextField.rx.text
             .asObservable()
-            .bind(to: cardNumberTextField.rx.text)
+            .replaceNilWith("")
+            .bind(to: viewModel.cardFullDigits)
```

Comment:
So here we see where we _were_ binding from `viewModel` -> user interface, but are now binding user interface -> `viewModel`, fixing the bug.